index.html

```diff
@@ -0,0 +1,177 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <header class="header-container">
+      <a href="/" class="logo-wrapper">
+        <img
+          src="/images/logo.webp"
+          alt="판다마켓 로고"
+          width="153"
+          height="51"
+        />
+      </a>
+      <a href="/login.html" class="button login">로그인</a>
+    </header>
+    <main>
+      <section class="main-bg-section-wrapper">
+        <div class="layout main-bg-contents-wrapper">
+          <div class="main-bg-contents-container">
+            <p class="main-bg-contents-description">
+              일상의 모든 물건을<br />거래해 보세요
+            </p>
+            <a href="/items.html" class="button visit">구경하러 가기</a>
+          </div>
+          <img
+            src="/images/header-img.webp"
+            alt="판다마켓 메인 헤더"
+            width="746"
+            height="340"
+          />
+        </div>
+      </section>
+      <section class="main-section-wrapper">
+        <div class="layout main-contents-wrapper">
+          <img
+            src="/images/contents-img-01.webp"
+            alt="판다마켓 메인 컨텐츠 핫이슈"
+            width="579"
+            height="444"
+          />
+          <div class="main-contents-container">
+            <p class="main-contents-title">Hot item</p>
+            <strong class="main-contents-strong"
+              >인기 상품을<br />확인해 보세요</strong
+            >
+            <p class="main-contents-description">
+              가장 HOT한 중고거래 물품을<br />판다 마켓에서 확인해 보세요
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="main-section-wrapper">
+        <div class="layout main-contents-wrapper">
+          <div class="main-contents-container left">
+            <p class="main-contents-title">Search</p>
+            <strong class="main-contents-strong"
+              >구매를 원하는 <br />상품을 검색하세요</strong
+            >
+            <p class="main-contents-description">
+              구매하고 싶은 물품은 검색해서 <br />쉽게 찾아보세요
+            </p>
+          </div>
+          <img
+            src="/images/contents-img-02.webp"
+            alt="판다마켓 메인 컨텐츠 서치"
+            width="579"
+            height="444"
+          />
+        </div>
+      </section>
+      <section class="main-section-wrapper">
+        <div class="layout main-contents-wrapper">
+          <img
+            src="/images/contents-img-03.webp"
+            alt="판다마켓 메인 컨텐츠 레지스터"
+            width="579"
+            height="444"
+          />
+          <div class="main-contents-container">
+            <p class="main-contents-title">Register</p>
+            <strong class="main-contents-strong"
+              >판매를 원하는 <br />상품을 등록하세요</strong
+            >
```

Comment:
💊 제안
현재 코드에서는 태그와 내부 텍스트가 들여쓰기 없이 한 줄로 작성되어 있어 가독성이 조금 떨어질 수 있습니다.
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
            <strong class="main-contents-strong">
              판매를 원하는 <br />상품을 등록하세요
            </strong>
```